### PR TITLE
Bootstrap development

### PR DESCRIPTION
**Pull request by mollygraton**
Adding bootstrap to page (for fun)

### CODE REVIEW

File: portfolio/src/main/webapp/style.css
Comment (cisxingg) [inline portfolio/src/main/webapp/style.css]: May apply here: new lines for each property(? I can't speak css...) https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation

**Review by cisxingg — APPROVED** [review]
I am not good enough at styling to give constructive feedback, so approval. Please get Warfa's approval before merging. I hope he could be more useful :D

File: portfolio/src/main/webapp/style.css
Comment (warfa-google) [inline portfolio/src/main/webapp/style.css]: Instead of putting margin-*: auto every where. You can set it 'margin:auto' once for all your pages in some sort of utility and then just re-set the specific margins you want to change from auto.
ao like margin:auto, and margin-top:5% instead of margin auto for all directions. You can also potentially do it in one line if you want.

**Review by warfa-google — APPROVED** [review]
Add the button class to the buttons on the comment page. Other than that LGTM!

**Pull request merged**
